runtime/imports.go

```diff
@@ -64,7 +64,8 @@ import (
 	"unsafe"
 
 	common "github.com/ChainSafe/gossamer/common"
-	"github.com/ChainSafe/gossamer/crypto"
+	"github.com/ChainSafe/gossamer/crypto/ed25519"
+	"github.com/ChainSafe/gossamer/crypto/sr25519"
```

Comment:
Should we include secp in runtime imports?

Reply:
we currently use the go-ethereum secp256k1 library in the imports, I am ok with leaving it for now